src/pudl/transform/eiaaeo.py

```diff
@@ -218,15 +321,7 @@ def _decumulate(df: pd.DataFrame, decumulate_columns: list[str]) -> pd.DataFrame
 
     # check that the totals all add up *before* decumulating, because
     # decumulating introduces some small amount more floating point error.
-    ratio_close_additions_to_total, ratio_close_reported_calculated = (
-        check_totals_add_up(unstacked)
-    )
-    assert (
-        0.999 < ratio_close_additions_to_total <= 1.0
-    ), f"planned + unplanned vs. total: {ratio_close_additions_to_total}"
-    assert (
-        0.999 < ratio_close_reported_calculated <= 1.0
-    ), f"reported vs. calculated totals: {ratio_close_reported_calculated}"
+    _check_totals_add_up(unstacked)
```

Comment:
Now that the useful functionality of "do the summing" is encapsulated, it seemed much easier to have a separate "make some assertions" function to do all the asset-specific assertions.